src/ReactiveSocket.cpp

```diff
@@ -241,15 +243,13 @@ void ReactiveSocket::createResponder(
         return;
       }
       std::shared_ptr<ChannelResponder> automaton;
-      ReactiveSocketSubscriberFactory subscriberFactory(
-          [&](folly::Executor* executor) {
-            ChannelResponder::Parameters params = {
-                {connection.shared_from_this(), streamId, handler},
-                executor ? *executor : defaultExecutor()};
-            automaton = std::make_shared<ChannelResponder>(params);
-            connection.addStream(streamId, automaton);
-            return automaton;
-          });
+      ReactiveSocketSubscriberFactory subscriberFactory([&](folly::Executor*) {
```

Comment:
this Factory thing will all go away, including the Executor argument. I didn't want to cause too much distracting code churn in this diff.